website/content/en/docs/kubebuilder/quickstart.md

```diff
@@ -0,0 +1,763 @@
+---
+title: Golang Based Operator Quickstart
+linkTitle: Quickstart
+weight: 1
+---
+
```

Comment:
For now, should we include a prominent note at the top saying something about this being an alpha feature, and maybe a link over to the main quickstart?